Tribler/Test/test_remote_torrent_handler.py

```diff
@@ -28,6 +28,8 @@ def setUpPreSession(self):
         super(TestRemoteTorrentHandler, self).setUpPreSession()
         self.config.set_dispersy(True)
         self.config.set_torrent_store(True)
+        self.config.set_megacache(True)
+        self.config.set_torrent_collecting(True)
```

Comment:
If we do not enable torrent_collecting, RemoteTorrentHandler will not be created, and there is no way to test it.